public/Player.js

```diff
@@ -1,16 +1,19 @@
 "use strict";
 //this class is going to create the players
+import PlayHand from "./PlayHand.js";
 
 export default class Player {
   domElement = null;
 
-  constructor(name, pot = 200, playerType = "GuestPlayer") {
+  constructor(confingObject) {
     this.hand = [];
```

Comment:
Why are the cards still being store in the `Player` and not in `PlayHand`? I see that only the first hand is stored by the `Player` and the second hand is stored in `PlayHand` which makes no sense, `PlayHand` should be the one holding the cards and the player deciding on how to split the cards and to which hand  to give a card to, `PlayHand` should know how to render the cards in its store and how to render them, count them and what state they are in if facing up/down